Vagrantfile

```diff
@@ -227,6 +227,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       v.memory            = vagrant_openshift_config['memory'].to_i
       v.cpus              = vagrant_openshift_config['cpus'].to_i
       v.customize ["modifyvm", :id, "--cpus", vagrant_openshift_config['cpus'].to_s]
+      full_provision(override.vm)
```

Comment:
I really don't want to add this step.  And I am guessing bigger changes would be necessary to make this the right change.  In general we have taken the mindset of keeping the images up to date rather than running provision on each vagrant up call.